src/app/footer-nav/cookies.component.html

```diff
@@ -735,4 +735,289 @@ <h3 class="heading-medium">Cookies used to make the service more secure</h3>
     </div>
   </ccd-accordion>
 
+  <!-- Family Public Law Accordion stuff -->
+  <ccd-accordion [title]="'Family Public Law'" [lastAccordion]="true">
+    <div class="accordion-body">
+
+      <!-- Family Public Law Start -->
+      <div class="para">
+        <span class="text-16">A cookie is a small piece of data that’s stored on your computer, tablet, or
+          phone when you visit a website. Most websites need cookies to work properly.</span>
+      </div>
+
+      <h3 class="heading-medium">How cookies are used in this service</h3>
+
+      <div class="para">
+        <span class="text-16">The service uses them to:</span>
+        <ul style="list-style-type: disc;">
+          <li>measure how you use the service so it can be improved</li>
+          <li>remember the notifications you've seen so that you're not shown them again</li>
+          <li>temporarily store the answers you give</li>
+        </ul>
+      </div>
+      <div class="para">
+        <span class="text-16">
+          Find out more about <a href="http://www.aboutcookies.org/managing-cookies" target="_blank">
+            how to manage cookies</a>.</span>
+      </div>
+
+      <h3 class="heading-medium">To measure website usage</h3>
+      <div class="para">
+        <span class="text-16">We use Google Analytics software to collect information about how you use this service.
+          We do this to help make sure the service is meeting the needs of its users and to help us make improvements,
+          for example improving site search.</span>
+      </div>
+      <div class="para">
+        <span class="text-16">Google Analytics stores information about:</span>
+        <ul style="list-style-type: disc;">
+          <li>the pages you visit</li>
+          <li>how long you spend on each page</li>
+          <li>how you got to the service</li>
+          <li>what you click on while you’re visiting the service.</li>
+        </ul>
+      </div>
+      <div class="para">
+        <span class="text-16">We allow Google to use or share our
+            analytics data. You can find out more about how Google use this information in their
+            <a [routerLink]="'/privacy-policy'" target="_blank">privacy policy</a>.</span>
```

Comment:
wrong privacy policy hyperlink

Reply:
I couldn't see any issue here.  Can you please elaborate?

Reply:
check back to the content doc attached to the story, You are supposed to hyperlink to the google privacy policy page not the moj link

https://policies.google.com/technologies/partner-sites?hl=en-GB&gl=uk